app/Helpers/helpers.php

```diff
@@ -67,28 +71,28 @@ function translate($value)
 {
     switch ($value) {
         case 'waiting':
-            return __('settings.book.waiting');
-            break;
+        return __('settings.book.waiting');
+        break;
         case 'reading':
-            return __('settings.book.reading');
-            break;
+        return __('settings.book.reading');
+        break;
         case 'returning':
-            return __('settings.book.returning');
-            break;
+        return __('settings.book.returning');
+        break;
         case 'returned':
-            return __('settings.book.returned');
-            break;
+        return __('settings.book.returned');
+        break;
         case config('view.notifications.waiting'):
-            return __('settings.notifications.waiting');
-            break;
+        return __('settings.notifications.waiting');
+        break;
         case config('view.notifications.returning'):
-            return __('settings.notifications.returning');
-            break;
+        return __('settings.notifications.returning');
+        break;
         case config('view.notifications.returned'):
-            return __('settings.notifications.returned');
-            break;
+        return __('settings.notifications.returned');
```

Comment:
**Reporter**: CHECKSTYLE</br>**Rule**: PSR2.ControlStructures.SwitchDeclaration.BreakIndent</br>**Severity**: error</br>**File**: app/Helpers/helpers.php</br>**Line**: 92</br>Terminating statement must be indented to the same level as the CASE body